drivers/backup/portworx/portworx.go

```diff
@@ -1535,6 +1617,166 @@ func (p *portworx) DeleteBackupSchedulePolicy(orgID string, policyList []string)
 	return nil
 }
 
+// CreateRuleForBackupWithMultipleApplications creates backup rule for multiple application in one rule
+func (p *portworx) CreateRuleForBackupWithMultipleApplications(orgID string, appList []string, ctx context.Context, appParameters ...map[string]backup.AppRule) (string, string, error) {
```

Comment:
Can we split this into 2 different functions - `CreatePreRuleForBackupWithMultipleApplications` and `CreatePostRuleForBackupWithMultipleApplications` ?
This is because here we are returning 2 strings and if an app does not have either pre or post rule, then how are we handling it? In the test case itself?